lib/cog/request.rb

```diff
@@ -8,7 +7,7 @@ class Request
     def initialize
       @args = populate_args
       @options = populate_options
-      @input = JSON.parse(STDIN.read)
+      @input = (STDIN.tty?) ? {} : JSON.parse(STDIN.read)
```

Comment:
If you want to pass something you'd do something like `echo '{"foo": "bar"}' | ./cog-command`. In that case, `STDIN` isn't attached to your terminal (it's coming in through the pipe), so everything still works.

What this does is remove the need to pipe in dummy information if you're not doing anything with the input.